frontend/src/components/phones/onboarding/RelayNumberConfirmationModal.tsx

```diff
@@ -0,0 +1,91 @@
+import { FormEventHandler, ReactElement, ReactNode, useRef } from "react";
+import { useLocalization } from "@fluent/react";
+import {
+  OverlayContainer,
+  FocusScope,
+  useDialog,
+  useModal,
+  useOverlay,
+  usePreventScroll,
+} from "react-aria";
+import { OverlayProps } from "@react-aria/overlays";
+import styles from "./RelayNumberConfirmationModal.module.scss";
+import { CloseIcon, InfoIcon } from "../../Icons";
+import { Button } from "../../Button";
+import { formatPhone } from "../../../functions/formatPhone";
+
+export type Props = {
+  onClose: () => void;
+  confirm: () => void;
+  relayNumber: string;
+};
+
+export const RelayNumberConfirmationModal = (props: Props) => {
+  const { l10n } = useLocalization();
+
+  const onSubmit: FormEventHandler = (event) => {
+    event.preventDefault();
+  };
+
+  return (
+    <OverlayContainer>
+      <PickerDialog
+        title={`test`}
```

Comment:
Forgot to use an actual title here :)

Reply:
removed as we don't use a title for this dialog